harbour-nettiradio.pro

```diff
@@ -1,46 +1,42 @@
 # The name of your application
 TARGET = harbour-nettiradio
 
-TEMPLATE = aux
 
-#CONFIG += sailfishapp
+CONFIG += sailfishapp_qml
 
 #SOURCES += src/harbour-nettiradio.cpp
 
-qml.files = qml/*
-
-qml.path = /usr/share/harbour-nettiradio/qml
-
-desktop.files = harbour-nettiradio.desktop
-desktop.path = /usr/share/applications
-
-OTHER_FILES += \
-    qml/* \
-    qml/Pages/* \
-    qml/Pages/StationLists/* \
-    qml/Pages/StationLists/Asemat.xml \
-    harbour-nettiradio.desktop \
-    rpm/harbour-nettiradio.spec \
-    #translations/*.ts \
-    rpm/harbour-nettiradio.changes
-
-# to disable building translations every time, comment out the
-# following CONFIG line
-# CONFIG += sailfishapp_i18n
-
-# TRANSLATIONS += translations/harbour-nettiradio-en.ts
-
-appicons.path = /usr/share/icons/hicolor/
-appicons.files = appicons/*
-
-INSTALLS += appicons qml desktop
-
 DISTFILES += \
     qml/Pages/StationLists/Asemat.xml \
     qml/Pages/StationLists/StationsModel.qml \
     qml/Pages/StationLists/xml/asemat.xml \
+    qml/Pages/About.qml \
+    qml/Pages/AlphaMenu.qml \
+    qml/Pages/AlpaMenuGroupView.qml \
+    qml/Pages/ContactNameGroup.qml \
+    qml/Pages/CoverPage.qml \
+    qml/Pages/FavDialog.qml \
+    qml/Pages/FavManager.qml \
+    qml/Pages/Help.qml \
+    qml/Pages/MainPage.qml \
+    qml/Pages/MediaButton.qml \
     qml/Pages/Menu.qml \
+    qml/Pages/PlayerPanel.qml \
+    qml/Pages/PulleyMenu.qml \
+    qml/Pages/SleepTimerPage.qml \
+    qml/Pages/StationGridView.qml \
+    qml/Pages/StationListView.qml \
     qml/Pages/functions.js \
-    qml/Pages/FavDialog.qml \
-    qml/Pages/FavManager.qml# \
-    #translations/harbour-nettiradio-en.ts
+    rpm/harbour-nettiradio.changes \
+    rpm/harbour-nettiradio.spec \
+    rpm/harbour-nettiradio.yaml \
+    harbour-nettiradio.desktop
+    # translations/harbour-nettiradio-en.ts
+
+SAILFISHAPP_ICONS = 86x86 108x108 128x128 172x172 #256x256
```

Comment:
What's with the icon sizes? I'm going to have to trust you on this one :)

Reply:
SAILFISHAPP_ICONS are there by default when creating new projects, so I added them. I think RPM complained about missing icons or something.